<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite errors="0" failures="1" hostname="xixi" name="pytest" skipped="0" tests="3" time="0.068" timestamp="2020-07-09T20:42:29.043386"><testcase classname="testing.test_calc" file="testing/test_calc.py" line="11" name="test_add" time="0.000"></testcase><testcase classname="testing.test_calc" file="testing/test_calc.py" line="17" name="test_add1" time="0.000"></testcase><testcase classname="testing.test_calc" file="testing/test_calc.py" line="22" name="test_div" time="0.001"><failure message="assert 1 == 2.0
  +1
  -2.0">@pytest.mark.div
    def test_div():
        cal = Calculator()
&gt;       assert 1 == cal.div(2, 1)
E       assert 1 == 2.0
E         +1
E         -2.0

test_calc.py:26: AssertionError</failure></testcase></testsuite></testsuites>